SplunkRumWorkspace/SplunkRum/SplunkRum/Session.swift

```diff
@@ -61,6 +61,9 @@ func getRumSessionId() -> String {
         isSessionIdChanged = true
         callbacks = sessionIdCallbacks
     }
+    if forceNewSessionId {
+        rumSessionId = generateNewSessionId()
+    }
```

Comment:
It seems we won't get a session ID change span if a new session ID is forced. Just trying to remember / understand what was the reasoning for this in our ticket. 😄 Feel free to merge if it's not important

Reply:
Asked Seerut about it, she said the session change span is expected

Reply:
@seemk I removed this if block and if we are forcing a new sessionId then it will go down the same path as when a sessionId expires.